app/models/county.rb

```diff
@@ -0,0 +1,13 @@
+class County < Area
+  def municipalities
+    self.zip_codes.map { |zip| zip.municipality }
+  end
+
+  def state
+    self.zip_codes.first.state
+  end
+
+  def name_abbreviated
```

Comment:
This should probably go in a `ViewModel`. Don't think I've added any yet so it's fine to keep here for now.

Reply:
I'll look into it.